docs/topics/readme/rest-http-secured-README.adoc

```diff
@@ -127,4 +128,6 @@ $ > GET /api/greeting HTTP/1.1
 > Accept: */*
 > Authorization: Bearer <TOKEN>
 ----
-====
\ No newline at end of file
+====
+
+$(integrationTestSection)
```

Comment:
Why add it twice in the same README?

Reply:
Honestly, It was a brain fart on my part:) Totally unintended. Removing the extra one at the bottom of the file. Thank you for spotting this.